.kitchen.yml

```diff
@@ -17,7 +17,6 @@ verifier:
 
 platforms:
   - name: ubuntu-16.04
-  - name: centos-7.3
```

Comment:
Need to put this back it. It was removed to make local testing a little quicker.